src/program/lwaftr/generate_binding_table/generate_binding_table.lua

```diff
@@ -43,8 +43,9 @@ end
 
 local function softwire_entry(v4addr, psid_len, b4)
    if tonumber(v4addr) then v4addr = to_ipv4_string(v4addr) end
-   return ("  softwire { ipv4 %s; psid %d; b4-ipv6 %s; }"):format(
-      v4addr, psid_len, b4)
+   local softwire = "  softwire { ipv4 %s; psid %d; b4-ipv6 %s; br-address %s;"
+   softwire = softwire .. " port-set { psid-length %d; }}"
+   return softwire:format(v4addr, psid_len, b4, br_address, port_set.psid_len)
```

Comment:
Did you mean to commit this?  This change applies only to lwaftr, not Snabb:wingo-next.